node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandlerTest.java

```diff
@@ -206,13 +209,28 @@ public void process_single_coredump_test() throws IOException {
         verify(coreCollector, never()).collect(any(), any());
         verify(coredumpReporter, times(1)).reportCoredump(eq("id-123"), eq("metadata"));
         assertFalse(Files.exists(coredumpDirectory));
-        assertFolderContents(doneCoredumpsPath, "id-123");
-        assertFolderContents(doneCoredumpsPath.resolve("id-123"), "metadata.json", "dump_bash.core.431.lz4");
+        assertFolderContents(doneCoredumpsPath.resolve("container-123"), "id-123");
+        assertFolderContents(doneCoredumpsPath.resolve("container-123").resolve("id-123"), "metadata.json", "dump_bash.core.431.lz4");
+    }
+
+    @Test
+    public void report_enqueued_and_processed_metrics() throws IOException {
+        Files.createFile(crashPathInContainer.resolve("dump-1"));
+        Files.createFile(crashPathInContainer.resolve("dump-2"));
+        Files.createFile(doneCoredumpsPath.resolve("container-123").resolve("dump-3"));
+
+        coredumpHandler.updateMetrics(context, crashPathInContainer);
+        List<DimensionMetrics> updatedMetrics = metrics.getMetricsByType(Metrics.DimensionType.PRETAGGED);
+        assertEquals(1, updatedMetrics.size());
+        Map<String, Number> values = updatedMetrics.get(0).getMetrics();
+        assertEquals(2, values.get("coredumps.enqueued").intValue());
+        assertEquals(1, values.get("coredumps.processed").intValue());
     }
 
     @Before
     public void setup() throws IOException {
-        Files.createDirectories(donePath);
+        Files.createDirectories(donePath.resolve("container-123"));
```

Comment:
I'd recommend using `UnixPath` in the test methods to easily create parent directories and/or files you need, the before method should only create directories are guaranteed to always exist, otherwise the tests might not catch that a certain directory is not created as expected.